src/commands/fun/8ball.js

```diff
@@ -4,7 +4,7 @@ import { eightball } from '../../data';
 import T from '../../translate';
 
 
-function eightBall(client, evt, suffix, lang) {
```

Comment:
Looking at the code over again, we can't remove `client` and `evt` as `suffix` and `lang` are passed last.